tests/behat/features/rest.feature

```diff
@@ -4,3 +4,4 @@ Feature: REST API
   Scenario: Check main menu REST endpoint
     When I do a GET request to "/entity/menu/main/tree"
     Then I should find in the position 1 of the menu the "title" with the value "Fundraise (Landing)"
+    Then I want to find the text "FAQ" in the JSON
```

Comment:
Can we make this the same as non-JSON syntax? i.e. `And I should see "FAQ"`

Reply:
Nop, is already taken.

Reply:
What about `"And I should see XXX in the JSON" ?`